src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/noc/nocworkflows/UpdateRepresentationWorkflowService.java

```diff
@@ -91,6 +91,16 @@ private boolean isOrganisationsEmpty(ChangeOrganisationRequest changeRequest) {
         return addedIsEmpty && removedIsEmpty;
     }
 
+    public boolean isNoApplicantOrganisationPolicy(Map<String, Object> caseData) {
+        return caseData.get(APPLICANT_ORGANISATION_POLICY) == null
+            || caseData.get(ORGANISATION_POLICY_ROLE).toString() == null;
```

Comment:
No need to use toString() here, it will throw null pointer exception if we add it